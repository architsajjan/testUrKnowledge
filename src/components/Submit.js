// IMPORTS
import React from 'react';

export default function Submit() {
    return (
        <div className="landing">
                { 
                   setTimeout(()=>{
                    history.pushState(null, null, location.href);
                    window.onpopstate = function() {
                        history.go(1);
                    };
                    }, 500)
                }
                <div className="">
                    <div className="instructions-main dark-overlay">
                        <h1>A copy of your result will be downloaded on successfull submission of your Test.</h1>
                    </div>
                </div>
                <div className="instructions-side">
                    <div className="container">
                        <h1 className="large text-primary">Note:</h1>
                        <p>
                            To go to the home page you could click the logo on navbar.
                        </p>
                    </div>    
                </div>
            </div>
        );
}

