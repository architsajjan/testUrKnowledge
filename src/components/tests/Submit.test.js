import React from "react";
import renderer from "react-test-renderer";

import Submit from '../Submit';

it("renders correctly", () => {
  const tree = renderer.create(<Submit />).toJSON();
  expect(tree).toMatchSnapshot();
});