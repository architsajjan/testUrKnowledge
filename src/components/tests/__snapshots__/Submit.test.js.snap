// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="landing"
>
  4
  <div
    className=""
  >
    <div
      className="instructions-main dark-overlay"
    >
      <h1>
        A copy of your result will be downloaded on successfull submission of your Test.
      </h1>
    </div>
  </div>
  <div
    className="instructions-side"
  >
    <div
      className="container"
    >
      <h1
        className="large text-primary"
      >
        Note:
      </h1>
      <p>
        To go to the home page you could click the logo on navbar.
      </p>
    </div>
  </div>
</div>
`;
